x-pack/plugins/security_solution/public/transforms/utils/create_indices_from_prefix.ts

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ELASTIC_NAME } from '../constants';
+
+export const createIndicesFromPrefix = ({
+  transformIndices,
+  prefix,
+}: {
+  transformIndices: string[];
+  prefix: string;
+}): string[] => {
+  return transformIndices.map((index) => `.${ELASTIC_NAME}_${prefix}_${index}`);
+};
```

Comment:
Would this make sense in `common` and shared between client/server for consistent index name generation?

Reply:
This is only used for the front end for security_solutions. Not the backend server side.

The metrics_entity plugin though has a similar utility that creates a prefix. I don't know how we share utils between plugins today with causing problems? Is there an easier way than pulling in 1 plugin into another plugin?

We might pull the plugin from metrics_entities into security_solutions, we just don't do that today right now.

Reply:
I will leave this here as-is for now.